api/pkg/apis/v1alpha1/providers/target/k8s/k8s_test.go

```diff
@@ -187,13 +187,13 @@ func TestDeploymentToComponents(t *testing.T) {
 	assert.Equal(t, 2, len(components))
 	assert.Equal(t, "evs", components[0].Name)
 	assert.Equal(t, "rocket", components[1].Name)
-	assert.Equal(t, "evaamscontreg.azurecr.io/evsclient:latest", components[0].Properties[model.ContainerImage])
+	assert.Equal(t, "prom/prometheus", components[0].Properties[model.ContainerImage])
 	assert.Equal(t, "evaamscontreg.azurecr.io/rocket:detection", components[1].Properties[model.ContainerImage])
```

Comment:
Please also replace this image `evaamscontreg.azurecr.io/rocket:detection`

Reply:
Thanks， I will replace it in next version

Reply:
I'm also wondering about `components[0].Name` or related fields. Should we also change them?